.guile

```diff
@@ -0,0 +1 @@
+dotguile
```

Comment:
Not sure why .guile is showing up here. This file does not exists in this dir and should not be here.